.github/CODEOWNERS

```diff
@@ -1,20 +1,43 @@
-.github/workflows/** @nf-core/a-team
-tests/config @nf-core/a-team
+.devcontainer/devcontainer.json @nf-core/infrastructure @nf-core/ops
+.editorconfig @nf-core/infrastructure @nf-core/ops
+.gitattributes @nf-core/infrastructure @nf-core/ops
+.github/** @nf-core/infrastructure @nf-core/ops
+.gitignore @nf-core/infrastructure @nf-core/ops
+.gitpod.yml @nf-core/infrastructure @nf-core/ops
+.nf-core.yml @nf-core/infrastructure @nf-core/ops
+.pre-commit-config.yaml @nf-core/infrastructure @nf-core/ops
+.prettierignore @nf-core/infrastructure @nf-core/ops
+.prettierrc.yml @nf-core/infrastructure @nf-core/ops
+.pytest_cache @nf-core/infrastructure @nf-core/ops
+.vscode @nf-core/infrastructure @nf-core/ops
+CITATION.cff @nf-core/infrastructure @nf-core/ops
+LICENSE @nf-core/infrastructure @nf-core/ops
+README.md @nf-core/infrastructure @nf-core/ops
+main.nf @nf-core/infrastructure @nf-core/ops
+nf-test.config @nf-core/infrastructure @nf-core/ops
+pytest.ini @nf-core/infrastructure @nf-core/ops
+ruff.toml @nf-core/infrastructure @nf-core/ops
```

Comment:
```suggestion
/* @nf-core/infrastructure @nf-core/ops
```

Reply:
Wont' that notify us on ALL files?

Reply:
Maybe just everything in the root directory? I think **/** would be everything?

Reply:
ok, so I put us on ** (which I believe should be everything)
And then no-one on modules/** and subworkflows/**
So that should do the trick